app/assets/stylesheets/style.scss

```diff
@@ -12,15 +13,20 @@ body {
     max-height: 100px;
   }
 }
+
 .scrollable{
-  width:150px;
-  max-width: 150px;
+  width:140px;
+  max-width: 140px;
   height:80px;
   max-height: 80px;
-  overflow-y: scroll;
+  overflow-y: auto;
   color: blue;
 }
-
+#event_show{
```

Comment:
Avoid using id selectors<br>Selector `event_show` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space